src/common/reset.css

```diff
@@ -36,8 +36,17 @@ address {
 
 .visuallyHidden {
   position: absolute !important;
-  height: 1px; width: 1px;
+  height: 1px;
+  width: 1px;
   overflow: hidden;
   clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
   clip: rect(1px, 1px, 1px, 1px);
 }
+
+.Parallax-outer-override {
```

Comment:
Since these classes refer to a specific package, can you please put them in `src/common/react-scroll-parallax.css` and include it in `normalize.css`?

Reply:
Sure, good idea.